src/ds/json_schema.h

```diff
@@ -43,7 +43,7 @@ namespace ds
     template <typename T>
     inline void fill_number_schema(nlohmann::json& schema)
     {
-      schema["type"] = "number";
+      schema["type"] = "integer";
```

Comment:
Not sure why this was done that way to begin with, it's not incorrect, but slightly less precise than it could be. https://cswr.github.io/JsonSchema/spec/basic_types/